cumulusci/robotframework/BaseLibrary.py

```diff
@@ -0,0 +1,42 @@
+import logging
+from robot.libraries.BuiltIn import BuiltIn, RobotNotRunningError
+from cumulusci.robotframework.utils import RetryingSeleniumLibraryMixin
+
+
+class BaseLibrary(RetryingSeleniumLibraryMixin):
+    ROBOT_LIBRARY_SCOPE = "GLOBAL"
+
+    def __init__(self, debug=False):
+        self.debug = debug
+        self.retry_selenium = True
+        # Turn off info logging of all http requests
+        logging.getLogger("requests.packages.urllib3.connectionpool").setLevel(
+            logging.WARN
+        )
+
+    @property
+    def salesforce_api_version(self):
+        try:
+            client = self.cumulusci.tooling
+            response = client._call_salesforce(
+                "GET", "https://{}/services/data".format(client.sf_instance)
+            )
+            latest_api_version = float(response.json()[-1]["version"])
+            return latest_api_version
+
+        except RobotNotRunningError:
+            # not sure if this should return None, a reasonable default value,
+            # or raise an exception. :-\
+            return None
```

Comment:
I think it's best to return None and let the caller decide whether they can do something reasonable as a default or not.

I think this might belong better in the CumulusCI library since that's the home for keywords that interact with the org via the API rather than selenium.